kubernetes_env/kube_env.py

```diff
@@ -51,10 +51,10 @@ def inject_istio():
 
 def deploy_addons():
     apply_cmd = "kubectl apply -f "
-    url = "https://raw.githubusercontent.com/istio/istio/release-1.8"
-    # cmd = f"{apply_cmd} {YAML_DIR}/prometheus-mod.yaml && "
-    # cmd = f"{apply_cmd} {url}/samples/addons/jaeger.yaml "
-    # cmd += f"{apply_cmd} {url}/samples/addons/grafana.yaml "
+    url = "https://raw.githubusercontent.com/istio/istio/release-1.9"
```

Comment:
We can probably improve this by having the addons we want to deploy as a list. We can do this in the next request.